tests/test_benchmark_simple_scenarios.py

```diff
@@ -52,6 +52,30 @@ def test_run_smoothly_json(self, margs):
         # make sure the sum of the bus flow is always zero (there are rounding errors)
         assert df_busses_flow.net_sum.map(lambda x: 0 if x < 1e-4 else 1).sum() == 0
 
+    # this ensure that the test is only ran if explicitly executed, ie not when the `pytest` command
+    # alone is called
+    @pytest.mark.skipif(
+        "tests/{}".format(fname) not in sys.argv, reason="requires python3.3"
+    )
+    @mock.patch("argparse.ArgumentParser.parse_args", return_value=argparse.Namespace())
+    def test_run_battery(self, margs):
```

Comment:
>  def test_run_battery(self, margs):

Maybe change to ´test_benchmark_AE_grid_battery´?

Reply:
Done in 6a636be88c63cb9b21a2a63acab308a63579198e